### DISCUSSION TITLE
what about a package?

### DISCUSSION THREAD

**Discussion by FrankEnderman** [Coding-Bunker/unofficial-discord-client#66] [Ideas] [answered]
I would like to see a compiled version of this app

**Comment by guerinoni** [comment, marked as answer]
For is very early stage and there isn't any precompiled package

**Reply by FrankEnderman** [reply]
oh thanks

**Reply by guerinoni** [reply]
Feel free to add any kind of supported pkg or feature, I'm very busy lately :')
